### PR TITLE
BETA: Register bo4.is-a.dev

### PR DESCRIPTION
**Pull request by hussain22g**
Added `bo4.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by hussain22g** [comment]
.

File: domains/ho2.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "ho2",
+        "email": "hussainaltahoo944@gmail.com",
+        "repo": "https://ho2.vercel.app/",
+        "description": "ho2 project"
+    },
+    "record": {
+        
+        "TXT": "vc-domain-verify=ho2.is-a.dev,e925e90a6f992a727337"
```
Comment (SX-9) [inline domains/ho2.json:10]: we do not support vercel (assuming since vc-domain-verify is a vercel thing when googled)

**Review by SX-9 — CHANGES_REQUESTED** [review]

**Review by hussain22g — COMMENTED** [review]
Ok, thanks, now I changed it to cf page

**Review by wdhdev — CHANGES_REQUESTED** [review]
We cannot allow these domains to be registered due to abuse risks.

**Pull request closed without merging**